### PR TITLE
Fix and test for invalid `operations` and `map` field types

### PR DESCRIPTION
**Pull request by MickVanDuijn**
When `operations` is of type `string`, and `operationsPath.set` is called, the error was not correctly propagated.

### CODE REVIEW

File: src/processRequest.mjs [hunk 1]
```diff
@@ -193,6 +197,14 @@ export const processRequest = (
               createError(413, `${maxFiles} max file uploads exceeded.`)
             )
 
+          if (typeof operations !== 'object')
```
Comment (jaydenseric) [inline src/processRequest.mjs]: I'm not sure this check is necessary, because `operations` has already been checked at this point?

**Comment by jaydenseric** [comment]
Thanks for bringing this up! I've handled a few issues (e.g. `null` is technically an object but should not be allowed for `operations`) and polished some things.

I'll take it from here, as the `map` field needs the same treatment. I've added a todo test.

**Comment by MickVanDuijn** [comment]
Thanks! This at least solved one of the problems I had, but null is another tricky one indeed! What would be the ETA for this to be in a new release?

**Comment by jaydenseric** [comment]
@MickVanDuijn I plan to release today.

**Comment by jaydenseric** [comment]
Thanks 🙌

**Comment by jaydenseric** [comment]
Published in [v8.0.5](https://github.com/jaydenseric/graphql-upload/releases/tag/v8.0.5) 🚀

**Comment by MickVanDuijn** [comment]
🚀  Thanks for the quick release!

**Pull request merged**